MoreLinq/TagFirstLast.cs

```diff
@@ -63,5 +63,34 @@ partial class MoreEnumerable
             return source.Index() // count-up
                          .CountDown(1, (e, cd) => resultSelector(e.Value, e.Key == 0, cd == 0));
         }
+
+        /// <summary>
+        /// Returns a sequence of tuples, where the N-th tuple contains the N-th
+        /// element of the source sequence and two booleans indicating whether the
+        /// element is the first and/or last.
+        /// </summary>
+        /// <typeparam name="TSource">The type of the elements of <paramref name="source"/>.</typeparam>
+        /// <param name="source">The source sequence.</param>
+        /// <returns>
+        /// Returns the resulting sequence.
+        /// </returns>
+        /// <remarks>
+        /// This operator uses deferred execution and streams its results.
+        /// </remarks>
+        /// <example>
+        /// <code><![CDATA[
+        /// var numbers = new[] { 123, 456, 789 };
+        /// var result = numbers.TagFirstLast();
+        /// ]]></code>
+        /// The <c>result</c> variable, when iterated over, will yield
+        /// <c>(123, <see langword="true"/>, <see langword="false"/>)</c>,
```

Comment:
Am, afraid this is still not correct. It needs to read plain as `<c>(123, True, False)</c>`. Ditto for the next two.

Reply:
You let it pass for the Zip method.
The diff here is the <c>..</c> tag.

Reply:
Hum ok : https://dotnetfiddle.net/YzoOgr

Reply:
> You let it pass for the Zip method.

Sorry if I did.